src/services/TestResultsService.ts

```diff
@@ -434,7 +434,7 @@ export class TestResultsService {
     let bool = false;
     if (payload.testTypes) {
       payload.testTypes.forEach((testType) => {
-        if ((this.isTestTypeAdr(testType) && testType.testResult === TEST_RESULT.PASS) && !testType.certificateNumber) {
+        if ((this.isTestTypeAdr(testType) && testType.testResult === TEST_RESULT.PASS) && payload.testStatus !== TEST_STATUS.CANCELLED && !testType.certificateNumber) {
```

Comment:
Unclear form story what this is doing. Can something be a testResult = pass AND cancelled?

Reply:
Apparently the answer is "yes"